### ISSUE TITLE
When committing multiline commit messages only the first *word* is surviving

### ISSUE THREAD

**Issue by coldcoff** [koog1000/vscode-fossil#21]
When committing multiline commit messages only the first *word* is surviving:

VSCode: (2-line commit message)
![image](https://user-images.githubusercontent.com/14311781/74718714-10fc4780-5233-11ea-859a-203d7dc3cf3b.png)

In fossil:
![image](https://user-images.githubusercontent.com/14311781/74718873-54ef4c80-5233-11ea-9c9a-5771c282466f.png)

**Comment by koog1000** [comment]
I'm not in front of a computer right now, can you create a test commit with multiple lines that does not use the colon ":" character?

**Comment by koog1000** [comment]
I tried this with multiple version of 2-line messages and it appears to work correctly.

Using as close to the same commit message you have:
VS Code:
![Selection_001](https://user-images.githubusercontent.com/28274937/74804753-feffc080-5295-11ea-87f6-8b5cb71e6c33.png)
Fossil:
![Selection_002](https://user-images.githubusercontent.com/28274937/74804840-438b5c00-5296-11ea-8fcb-31c920936b9c.png)

Can you provide more details about how to reproduce this error?

**Comment by coldcoff** [comment]
Thank you @koog1000.

I played with it the whole morning and found the issue!
It was *here*, with *my* setup. Sorry for the noise and the false alarm!

I have a thin wrapper script around fossil (that I had forgotten about) which finally does
```bash
...
exec $HOME/inst/fossil/fossil $*
```
This needed to be changed to
```bash
...
exec $HOME/inst/fossil/fossil "$@"
```

-> Now the calls from VSCode work as expected!

Interesting enough when using fossil from the commandline -- as I do since years -- I never had issues with `$*`.

Closing issue.
